pages/projects.tsx

```diff
@@ -1,17 +1,21 @@
 import next from 'next';
-import React from 'react';
+import React, { useState } from 'react';
 import Description from '../components/description/Description';
 import Navbar from '../components/layout/navbar/navbar';
 import style from '../styles/projects.module.scss';
 import ProjectPage from '../components/projectPage/projectPage';
 import Footer from '../components/layout/footer/footer';
 import Layout from '../components/layout/layout';
+
 export default function vast() {
+  const [currentHeight, setCurrentHeight] = useState(0);
+  console.log(currentHeight);
+
   return (
     <>
-      <Navbar />
-      <Description descriptionOutput={<ProjectPage />} />
-      <Footer />
+      <Layout descriptionText={ProjectPage()}>
```

Comment:
If you have only one sibling in the root of the component template, you can remove the fragment.

Reply:
Done
tnx, learned for the future (: